src/fragments/lib/datastore/js/getting-started/30_platformIntegration.mdx

```diff
@@ -56,5 +56,40 @@ DataStore.configure({
 });
 ```
 
+</Block>
+<Block name="Expo">
+
+Start with the [Expo](https://docs.expo.dev/):
+
+```bash
+expo init AmplifyDataStoreExpo
+cd AmplifyDataStoreExpo
+npx amplify-app@latest
+npm install @react-native-community/netinfo @react-native-async-storage/async-storage
```

Comment:
Should this be `expo install`?

Reply:
Yes, we should use `expo install`. So, it uses the compatible versions with the Expo SDK.